src/molecules/StarRating/__tests__/star_rating.spec.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import renderer from 'react-test-renderer';
+
+import StarRating from 'molecules/StarRating';
+
+describe('<StarRating />', () => {
+
+  it('renders correctly', () => {
+    const wrapper = mount(<StarRating />);
+
+    expect(wrapper.type()).toEqual(StarRating);
+  });
+
+  it('renders correctly', () => {
+    const actual = renderer.create(<StarRating />).toJSON();
```

Comment:
It's not essential for right now, and maybe leave this for another pull request, but it would be good to have these tests include the props that you'd expect.